src/Models/Post.php

```diff
@@ -1,18 +1,70 @@
 <?php
```

Comment:
I'd like to keep this file as it was, as well as removing the IDE helper. It's overkill to me.